generate.sh

```diff
@@ -98,97 +98,110 @@ fi
 
 mkdir yaydoctemp
 BUILD_DIR=$(pwd)/yaydoctemp
+TEMP_NAME=yaydoctemp
+function generate {
 
-cd $ROOT_DIR
+  cd $1
```

Comment:
Store the positional args in named variables or add comment explaining each positional args